vavr/src/test/java/io/vavr/collection/AbstractSeqTest.java

```diff
@@ -140,14 +140,14 @@ public void shouldNarrowSeq() {
 
     @Test
     public void shouldAppendElementToNil() {
-        final Seq<Integer> actual = this.<Integer> empty().append(1);
+        final Seq<Integer> actual = this.<Integer>empty().append(1);
```

Comment:
Was it necessary to change the formatting of the code?
Formatting changes a hide the real changes and not necessary as I think.

Reply:
It was not done on purpose, I used Ctrl + Alt + L and these changes were performed

Reply:
@valery1707 do you think I should change it back?

Reply:
@NataliiaPrivezentseva, I think that you should not change formatting of lines that you don't really change.